src/EditorFeatures/Core/Implementation/Suggestions/SuggestionsOptions.cs

```diff
@@ -10,10 +10,10 @@ internal static class SuggestionsOptions
     {
         private const string FeatureName = "SuggestionsOptions";
 
-        public static readonly Option2<bool?> Asynchronous = new(FeatureName, nameof(Asynchronous), defaultValue: null,
+        public static readonly Option2<bool> Asynchronous = new(FeatureName, nameof(Asynchronous), defaultValue: true,
             new RoamingProfileStorageLocation("TextEditor.Specific.Suggestions.Asynchronous2"));
 
-        public static readonly Option2<bool> AsynchronousFeatureFlag = new(FeatureName, nameof(AsynchronousFeatureFlag), defaultValue: false,
-            new FeatureFlagStorageLocation("Roslyn.AsynchronousQuickActions"));
+        public static readonly Option2<bool> AsynchronousQuickActionsDisableFeatureFlag = new(FeatureName, nameof(AsynchronousQuickActionsDisableFeatureFlag), defaultValue: false,
```

Comment:
Consider having a tracking item or a comment discussing when we might want to sunset this. Writing a comment of "we should get rid of this by September 2022" obviously makes zero guarantee that'll actually happen, but it means somebody five years from now knows this can probably go.